### PR TITLE
Additional section for the search span

### PR DESCRIPTION
**Pull request by mervebolatdd**
Adding a new section to explain a feature enhancement to trace view.

### Merge instructions

- [ ] Please merge after reviewing

### CODE REVIEW

**Review by assma-ben-aicha — APPROVED** [review]

File: content/en/tracing/trace_explorer/trace_view.md [hunk 1]
```diff
@@ -93,6 +93,35 @@ Hover over a service to highlight its parent and children, and click on it to fo
 {{% /tab %}}
 {{< /tabs >}}
 
+## Span search
+
+In the Waterfall and Flamegraph visualizations, the search option allows you to quickly find the spans that meet specific queries. Spans that match the search query are highlighted in the trace view and you can navigate between these matches using the arrows next to the search bar.
```
Comment (buraizu) [inline content/en/tracing/trace_explorer/trace_view.md:98]:
```suggestion
In the Waterfall and Flamegraph visualizations, the search option allows you to find the spans that meet specific queries. Spans that match the search query are highlighted in the trace view and you can navigate between these matches using the arrows next to the search bar.
```

File: content/en/tracing/trace_explorer/trace_view.md [hunk 2]
```diff
@@ -93,6 +93,35 @@ Hover over a service to highlight its parent and children, and click on it to fo
...
+
+<img src="https://github.com/user-attachments/assets/4c6fc35a-65b7-42ee-bc24-fcb9adb9d0b8" alt="Search bar" style="width:90%;">
+
+Note: When the `Error` checkbox is selected, the search results return spans that match the query and are in an error state.
```
Comment (buraizu) [inline content/en/tracing/trace_explorer/trace_view.md:102]:
```suggestion
**Note**: When the `Error` checkbox is selected, the search results return spans that match the query and are in an error state.
```

File: content/en/tracing/trace_explorer/trace_view.md [hunk 3]
```diff
@@ -93,6 +93,35 @@ Hover over a service to highlight its parent and children, and click on it to fo
...
+
+The search query on trace side panel supports the following options:
```
Comment (buraizu) [inline content/en/tracing/trace_explorer/trace_view.md:104]:
```suggestion
The search query on the trace side panel supports the following options:
```

File: content/en/tracing/trace_explorer/trace_view.md [hunk 4]
```diff
@@ -93,6 +93,35 @@ Hover over a service to highlight its parent and children, and click on it to fo
...
+
+**Free text search:**
+Free-form text search allows filtering by service, resource or operation name. It highlights the spans containing the specified text within these categories.
```
Comment (buraizu) [inline content/en/tracing/trace_explorer/trace_view.md:107]:
```suggestion
Free-form text search allows filtering by service, resource, or operation name. It highlights the spans containing the specified text within these categories.
```

File: content/en/tracing/trace_explorer/trace_view.md [hunk 5]
```diff
@@ -93,6 +93,35 @@ Hover over a service to highlight its parent and children, and click on it to fo
...
+Example: `web`
+
+**Key-value search:**
+Use key:value expression to filter spans with specific key-value pairs.
+Example: `service:web-ui`
+
+Note: Wildcard is not supported for Trace search bar.
```
Comment (buraizu) [inline content/en/tracing/trace_explorer/trace_view.md:114]:
```suggestion
**Note**: Wildcards are not supported in the Trace search bar.
```

File: content/en/tracing/trace_explorer/trace_view.md [hunk 6]
```diff
@@ -93,6 +93,35 @@ Hover over a service to highlight its parent and children, and click on it to fo
...
+
+**Supported experssions:**
+Group expression: `language:(go OR python)`
+Boolean expression: `service: event-query OR terminator`
```
Comment (buraizu) [inline content/en/tracing/trace_explorer/trace_view.md:118]:
```suggestion
Boolean expression: `service:event-query OR terminator`
```

File: content/en/tracing/trace_explorer/trace_view.md [hunk 7]
```diff
@@ -93,6 +93,35 @@ Hover over a service to highlight its parent and children, and click on it to fo
...
+Range expression: `duration:>200ms`
+
+
+Note: Numberical values suport `<`, `>`, `<=`, and `>=` expressions.
```
Comment (buraizu) [inline content/en/tracing/trace_explorer/trace_view.md:122]:
```suggestion
**Note**: Numerical values support `<`, `>`, `<=`, and `>=` expressions.
```

**Review by buraizu — CHANGES_REQUESTED** [review]
Thanks for the PR, just requesting some minor edits.

**Review by mervebolatdd — COMMENTED** [review]
Thanks for the review

**Review by buraizu — APPROVED** [review]
@mervebolatdd Thank you for applying those changes, this looks good to me and is approved. Let me know either here or in the documentation Slack channel when this PR is ready to be merged.

**Pull request merged**